pandas/core/frame.py

```diff
@@ -1697,19 +1697,42 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
 
         .. versionadded:: 0.21.0
 
+        This function writes the dataframe as a parquet file. You
+        can choose different parquet backends, and have the option
+        of compressing.
+
         Parameters
         ----------
         fname : str
-            string file path
+            String file path.
         engine : {'auto', 'pyarrow', 'fastparquet'}, default 'auto'
             Parquet library to use. If 'auto', then the option
             ``io.parquet.engine`` is used. The default ``io.parquet.engine``
             behavior is to try 'pyarrow', falling back to 'fastparquet' if
             'pyarrow' is unavailable.
         compression : {'snappy', 'gzip', 'brotli', None}, default 'snappy'
             Name of the compression to use. Use ``None`` for no compression.
-        kwargs
-            Additional keyword arguments passed to the engine
+        kwargs : dict
```

Comment:
can you change this line to `**kwargs` (without the dict, as you actually cannot pass it as a dict)

Reply:
Writing it as **kwargs results in a validation error: 

```
Errors found:
	Errors in parameters section
		Parameters {'kwargs'} not documented
		Unknown parameters {'**kwargs'}
		Parameter "**kwargs" has no type
```

Reply:
You can ignore that error (the validation script is not yet perfect here :-))

Reply:
Thanks for clarifying!